src/board/system76/common/acpi.c

```diff
@@ -115,15 +115,17 @@ uint8_t acpi_read(uint8_t addr) {
 
         ACPI_8(0xCC, sci_extra);
 
-        ACPI_8(0xCE, DCR2);
-#if HAVE_CPU_FAN2
-        ACPI_8(0xCF, DCR3);
-#endif
+        ACPI_8(0xCE, PWM_REG(CPU_FAN1));
         ACPI_8(0xD0, F1TLRR);
         ACPI_8(0xD1, F1TMRR);
+#ifdef CPU_FAN2
```

Comment:
What happens if CPU_FAN2 and HAVE_DGPU is defined at the same time?

Should we add :

```C
#if defined(CPU_FAN2) && defined(HAVE_DGPU)
#error "CPU_FAN2 and HAVE_DGPU are mutually exclusive"
#endif
```

Reply:
done in https://github.com/Dasharo/ec/pull/70/commits/97f29457adbd1eea0872cf9cf638a60804abd45f

Reply:
@miczyg1 ping